### DISCUSSION TITLE
Add Prefixes for staffmsg:

### DISCUSSION THREAD

**Discussion by Gintamier** [Zrips/CMI#3362] [Feature Requests]
**Description of feature/improvement request:**
It would be nifty and nice to be able to see the ranks of users that are talking within the /staffmsg

```yml
  staffmsg:
    help:
      info: '&eSends message to staff chanell'
      args: '[message/toggle/on/off]'
    info:
      message: '&2[Staff]&4[&7[senderDisplayName]&4]&r&c [message]'
      toggledOn: '&ePersistent staff chat have been started. You can now write messages
        in public chat.'
      toggledOff: '&eToggled off Persistent staff chat'
```
---
### Basic explanation as to why this feature is needed and/or useful to have: ###
It is useful to be able to see what rank the staff member is that you are talking to,
Without having to needlesly check the tablist

**I have been aware of it works with some permission plugins, but it does not work with Ultra Permissions.**

Seems that CMI is not able to read Uperms Prefixes.

**Comment by Zrips** [comment]
uperms seams to be supported. you can always try using [senderPrefix] and add prefix to the locale line if you want to have something like that
